cmd/crc/cmd/status.go

```diff
@@ -59,23 +71,30 @@ func runStatus() {
 		return err
 	})
 	if err != nil {
-		errors.ExitWithMessage(1, "Error finding size of cache: %s", err.Error())
+		return fmt.Errorf("Error finding size of cache: %s", err.Error())
 	}
 	cacheUsage := units.HumanSize(float64(size))
 	diskUse := units.HumanSize(float64(clusterStatus.DiskUse))
 	diskSize := units.HumanSize(float64(clusterStatus.DiskSize))
 	diskUsage := fmt.Sprintf("%s of %s (Inside the CRC VM)", diskUse, diskSize)
 	status := Status{clusterStatus.CrcStatus, clusterStatus.OpenshiftStatus, diskUsage, cacheUsage, cacheDir}
-	printStatus(status, statusFormat)
+
+	if outputFormat == jsonFormat {
```

Comment:
this is more generic, so the writer is better suited to be created in a generic way as ALL commands will use this.

Reply:
I prefer to start small and don't over-engineer it in this PR. But for sure, when we will have 2 commands with `-o json` we will need to do something.